### ISSUE TITLE
publish for Scala 2.13.0-M5?

### ISSUE THREAD

**Issue by SethTisue** [suzaku-io/boopickle#113]
this is needed by http4s over at https://github.com/http4s/http4s/pull/2393

**Comment by rossabaker** [comment]
I looked for a quick win, but native is going to cause some challenges:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scala-native#nativelib_native0.3_2.13.0-M5;0.3.8: not found
[warn] 	:: org.scala-native#javalib_native0.3_2.13.0-M5;0.3.8: not found
[warn] 	:: org.scala-native#auxlib_native0.3_2.13.0-M5;0.3.8: not found
[warn] 	:: org.scala-native#scalalib_native0.3_2.13.0-M5;0.3.8: not found
[warn] 	:: org.scala-native#test-interface_native0.3_2.13.0-M5;0.3.8: not found
[warn] 	:: com.lihaoyi#utest_native0.3_2.13.0-M5;0.6.6: not found
[warn] 	:: org.scala-native#nscplugin_2.13.0-M5;0.3.8: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by SethTisue** [comment]
the 2.13/SN should just be excluded from the matrix, I think?

**Comment by mfirry** [comment]
2.13.0-RC1 is out now

**Comment by SethTisue** [comment]
this is blocking http4s in the Scala 2.13 community build

**Comment by rossabaker** [comment]
http4s has a [thirteen configuration](https://github.com/http4s/http4s/blob/63e999f4edafab5aaa4e0a301ed650b1d97a0873/.travis.yml#L44) to work around this while we wait, but I'm not sure how that would tie into dbuild.  Would an `extra.exclude` for boopickle not do the trick?

**Comment by ochrons** [comment]
1.3.1 has been released with 2.13-RC1 support, give it a try @rossabaker!
